### ISSUE TITLE
Turn on anomaly forcing for ISSP compsets

### ISSUE THREAD

**Issue by ekluzek** [ESCOMP/CTSM#922]
Automatically turn on the anomaly forcing when ISSP compsets are being used. This should go on both release-clm5.0 and master.

The primary change is in cime, which should go on maint-5.6 as well as master.

ESMCI/cime/#3391

Periodically changes in cime from maint-5.6 are automatically moved to master.

**Comment by wwieder** [comment]
link to #987

**Comment by ekluzek** [comment]
I don't think we should do this on the release-clm5.0 branch anymore. But, we do want it on main development still.

**Comment by wwieder** [comment]
Yes, I think we want this feature available on the development branch.
Thanks Erik
